PrivateCloud.DiagnosticInfo/PrivateCloud.DiagnosticInfo.psm1

```diff
@@ -42,12 +42,12 @@ $CommonFunc = {
     #
     # Show arbitrary normal status message, with optional color coding
     #
-    function Show-Update(

```

Comment:
Please re-merge - no changes here?

Reply:
I have no idea how this got in here and why git believes that there are changes.